managed/ui/src/components/schedules/Schedules.js

```diff
@@ -106,6 +119,9 @@ class ScheduleDisplayItem extends Component {
             <DescriptionItem title={'Schedule (UTC)'}>
               <span>{schedule.frequency || schedule.cronExpression}</span>
             </DescriptionItem>
+            <DescriptionItem title={'Backup Type'}>
```

Comment:
`title='Backup Type'`. No need for curly braces.

Reply:
Done